src/microbe_stage/Microbe.cs

```diff
@@ -798,8 +798,7 @@ public void Kill()
             // If no organelles have a scene, use mitochondrion as fallback
             if (sceneToUse.LoadedScene == null)
             {
-                sceneToUse.LoadedScene = SimulationParameters.Instance.GetOrganelleType(
-                    "mitochondrion").LoadedScene;
+                sceneToUse.LoadedScene = GD.Load<PackedScene>("res://assets/models/organelles/Cytoplasm.tscn");
```

Comment:
While this would exactly work with the requirement of showing the cytoplasm scene, I'd like to see a more general approach:
- Add a new property to the OrganelleDefinition like CorpseChunkScene
- Load that when the other organelle scenes are loaded
- Add a check to the OrganelleDefinition that either the DisplayScene is not null or the CorpseChunkScene is not null
- Add detection on line 790 logic to use that CorpseChunkScene if that is not null and the DisplayScene was null

This way the system will be extensible if we ever need to have something else. And finally add an exception throw here saying that organelle definitions are invalid as it is no longer possible to reach a situation where sceneToUse is null.